mpas_analysis/ocean/meridional_overturning_circulation.py

```diff
@@ -283,6 +283,7 @@ def _compute_moc_climo_postprocess(config, runStreams, variableMap, calendar,
                        dictClimo['endYearClimo'])
     if not os.path.exists(outputFileClimo):
         print '   Load data...'
+        chunking = config.getExpression(sectionName, 'maxChunkSize')
```

Comment:
Please see my edit @milenaveneziani.  Effectively we don't have the flexibility of the config parser to return `None` as default for a config option that is intended to be integer.  This was my original idea but it was not readily supported by the config parser.  The simpler approach is to use the new commit that returns `None` if the config option is not available, which is effectively another way of returning a default from `config.getExpression`.  I also added one more test to demonstrate we get an integer with integer input from `config.getExpression`.  Additional implicit type casting rules are demonstrated via new unit tests.